### PR TITLE
Merging 2 recipes to one recipe WebSphereUnavailableSSOMethods

### PR DESCRIPTION
**Pull request by ranuradh**
## What's changed?
This recipe merges WebSphereUnavailableSSOTokenMethod and WebSphereUnavailableSSOCookieMethod into one recipe WebSphereUnavailableSSOMethods

## What's your motivation?
They were separate recipes and were resulting in conflicts because they were both working on the same class com.ibm.websphere.security.WSSecurityHelper

## Anyone you would like to review specifically?
@cjobinabo 

### Checklist
- [X] I've added unit tests to cover both positive and negative cases
- [X] I've read and applied the [recipe conventions and best practices](https://docs.openrewrite.org/authoring-recipes/recipe-conventions-and-best-practices)
- [X] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

File: src/main/resources/META-INF/rewrite/was-to-liberty.yml [hunk 1]
```diff
@@ -26,13 +26,30 @@ description: >
...
+      methodPattern: com.ibm.websphere.security.WSSecurityHelper getLTPACookieFromSSOToken()
+      newMethodName: getSSOCookieFromSSOToken
+      ignoreDefinition: true
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.ibm.websphere.security.WSSecurityHelper
+      newFullyQualifiedTypeName: com.ibm.websphere.security.web.WebSecurityHelper
+      ignoreDefinition: true
 ---
 type: specs.openrewrite.org/v1beta/recipe
 name: org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod
```
Comment (cjobinabo) [inline src/main/resources/META-INF/rewrite/was-to-liberty.yml:55]: It looks like the `org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod` recipes still needs to be removed.

File: src/test/java/org/openrewrite/java/liberty/WebSphereUnavailableSSOMethodsTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.config.Environment;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class WebSphereUnavailableSSOMethodsTest implements RewriteTest {
```
Comment (cjobinabo) [inline src/test/java/org/openrewrite/java/liberty/WebSphereUnavailableSSOMethodsTest.java:28]: Can you also provide a test with both issues need to be resolved so we can validate the recipes execute in the correct order.

File: src/main/resources/META-INF/rewrite/was-to-liberty.yml (hunk 1 above)
Comment (ranuradh) [reply src/main/resources/META-INF/rewrite/was-to-liberty.yml:55]: It's a custom recipe. so wasnt sure how I could remove it

Comment (ranuradh) [reply src/main/resources/META-INF/rewrite/was-to-liberty.yml]: done in next commit

File: src/test/java/org/openrewrite/java/liberty/WebSphereUnavailableSSOMethodsTest.java (hunk 2 above)
Comment (ranuradh) [reply src/test/java/org/openrewrite/java/liberty/WebSphereUnavailableSSOMethodsTest.java]: the test is updated with both the methods

File: src/main/resources/META-INF/rewrite/was-to-liberty.yml [hunk 3]
```diff
@@ -26,21 +26,22 @@ description: >
...
 ---
 type: specs.openrewrite.org/v1beta/recipe
-name: org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod
-displayName: Use `getSSOCookieFromSSOToken`
+name: org.openrewrite.java.liberty.WebSphereUnavailableSSOMethods
+displayName: Use `getSSOCookieFromSSOToken` and `logout`
 description: >
-  This method `LTPACookieFromSSOToken()` is deprecated in traditional WebSphere Application 
-  Server Version 8.5 and might be removed in a future release. It is not available on Liberty.
+  This recipe replaces`LTPACookieFromSSOToken()` with  `getSSOCookieFromSSOToken`  and `revokeSSOCookies` with `logout`. The two methods are  deprecated in traditional WebSphere Application
```
Comment (cjobinabo) [inline src/main/resources/META-INF/rewrite/was-to-liberty.yml:41]: We need to add a space between `replacesLTPACookieFromSSOToken()`. Aside from that everything else looks good

Comment (ranuradh) [reply src/main/resources/META-INF/rewrite/was-to-liberty.yml]: added in next commit

Comment (cjobinabo) [reply src/main/resources/META-INF/rewrite/was-to-liberty.yml]: Thanks!

**Review by cjobinabo — APPROVED** [review]

**Pull request merged**